portal/forms/home.py

```diff
@@ -36,10 +36,11 @@
 # identified as the original program.
 from django import forms
 from portal.helpers.regex import get_regex_name, get_regex_message, get_regex_telephone
-import re
+from portal.helpers import captcha
 
 
 class ContactForm(forms.Form):
+
```

Comment:
I think PEP 8 validation gets upset about certain blank lines, but I can't remember if this is one of them... Either way, you'd usually not leave one here.